components/nav/Navbar.vue

```diff
@@ -1,10 +1,19 @@
 <script setup lang="ts">
-
+//TODO: add animation, maybe with https://www.impressivewebs.com/animate-display-block-none/
 </script>
 
 <template>
-  <UContainer class="flex align-center w-full h-16 border-b border-neutral-300 dark:bg-gray-900 dark:border-gray-800">
-    <NavBarThemeToggle />
-    <CalendarInfoButton />
+  <UContainer class="flex underline-offset-1 h-30">
```

Comment:
`class="flex underline-offset-1 w-full h-30 border-b border-neutral-300 dark:bg-gray-900 dark:border-gray-800"`

<img width="1556" alt="image" src="https://github.com/Dino-Kupinic/UntisPlanner/assets/87943435/2ba4529b-3bd7-43f2-a70a-651a988c2513">

We should re-add this to keep a more consistent look